File: src/main/java/com/example/storage/repository/EmployeeRepository.java
package com.example.storage.repository;

import com.example.storage.entity.Employee;
import org.springframework.data.jpa.repository.EntityGraph;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Optional;

public interface EmployeeRepository extends JpaRepository<Employee, Long> {
    @EntityGraph(attributePaths = {"issueList", "issueList.equipment", "issueList.equipment.category"})
    Optional<Employee> findByCode(String code);
}

File: src/main/kotlin/ru/nto/storage/repository/EmployeeRepository.kt
package ru.nto.storage.repository

import org.springframework.data.jpa.repository.EntityGraph
import org.springframework.data.jpa.repository.JpaRepository
import ru.nto.storage.entity.Employee

interface EmployeeRepository : JpaRepository<Employee, Long> {
    @EntityGraph(attributePaths = ["issueList", "issueList.equipment", "issueList.equipment.category"])
    fun findByCode(code: String): Employee?
}
